prep-css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc( 100%/6);
```

Comment:
`flex-basis` only works when it is applied to children of elements with `display: flex;` try moving all of your `flex-basis` properties to the rule set for `.card-wrapper`.

Reply:
![image](https://user-images.githubusercontent.com/106715786/173544668-a0422afc-609c-4bdc-98f0-9fdbe2a4a6a9.png)

Reply:
![image](https://user-images.githubusercontent.com/106715786/173545460-0d64c923-0b98-4e78-af3c-7d4da022b9af.png)